pkg/templates/validatingwebhook.go

```diff
@@ -0,0 +1,31 @@
+package templates
+
+import (
+	admrv1 "k8s.io/api/admissionregistration/v1"
+	"k8s.io/utils/ptr"
+)
+
+var SubscriptionValidatingWebhook = admrv1.ValidatingWebhook{
+	Name: "subscription.ocs.openshift.io",
```

Comment:
Names (or namespaces) should not be incorporated into templates
```suggestion
```

Reply:
done.